doc/reference/widgets/ChatInterface.md

```diff
@@ -0,0 +1,207 @@
+```python
```

Comment:
I would like to add an example of using the `FileInput`. And example like below raises an exception when I upload and send a csv file.

https://datasets.holoviz.org/penguins/v1/penguins.csv

```python
import panel as pn
from panel.widgets import ChatInterface

pn.extension()

ChatInterface(
    widgets=[pn.widgets.FileInput(name="CSV File", accept=".csv")]
).servable()
```

```bash
$ panel serve script.py --autoreload --index script
2023-08-16 05:50:04,450 Starting Bokeh server version 3.1.1 (running on Tornado 6.3.2)
2023-08-16 05:50:04,451 User authentication hooks NOT provided (default user enabled)
2023-08-16 05:50:04,463 Bokeh app running at: http://localhost:5006/
2023-08-16 05:50:04,463 Bokeh app running at: http://localhost:5006/script
2023-08-16 05:50:04,463 Starting Bokeh server with process id: 11124
2023-08-16 05:50:11,539 WebSocket connection opened
2023-08-16 05:50:11,540 ServerConnection created
2023-08-16 05:50:20,736 error handling message
 message: Message 'PATCH-DOC' content: {'events': [{'kind': 'MessageSent', 'msg_type': 'bokeh_event', 'msg_data': {'type': 'event', 'name': 'button_click', 'values': {'type': 'map', 'entries': [['model', {'id': 'p1062'}]]}}}]} 
 error: AttributeError("'NoneType' object has no attribute 'css_classes'")
Traceback (most recent call last):
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/server/protocol_handler.py", line 97, in handle
    work = await handler(message, connection)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/server/session.py", line 94, in _needs_document_lock_wrapper
    result = func(self, *args, **kwargs)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/server/session.py", line 288, in _handle_patch
    message.apply_to_document(self.document, self)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/protocol/messages/patch_doc.py", line 104, in apply_to_document
    invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.payload, setter=setter))
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/callbacks.py", line 443, in invoke_with_curdoc
    return f()
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/protocol/messages/patch_doc.py", line 104, in <lambda>
    invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.payload, setter=setter))
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/document.py", line 376, in apply_json_patch
    DocumentPatchedEvent.handle_event(self, event, setter)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/events.py", line 246, in handle_event
    event_cls._handle_event(doc, event)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/events.py", line 281, in _handle_event
    cb(event.msg_data)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/callbacks.py", line 390, in trigger_event
    model._trigger_event(event)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/util/callback_manager.py", line 113, in _trigger_event
    self.document.callbacks.notify_event(cast(Model, self), event, invoke)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/callbacks.py", line 260, in notify_event
    invoke_with_curdoc(doc, callback_invoker)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/document/callbacks.py", line 443, in invoke_with_curdoc
    return f()
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/bokeh/util/callback_manager.py", line 109, in invoke
    cast(EventCallbackWithEvent, callback)(event)
  File "/home/jovyan/repos/private/panel/panel/reactive.py", line 492, in _server_event
    self._comm_event(doc, event)
  File "/home/jovyan/repos/private/panel/panel/reactive.py", line 479, in _comm_event
    state._handle_exception(e)
  File "/home/jovyan/repos/private/panel/panel/io/state.py", line 436, in _handle_exception
    raise exception
  File "/home/jovyan/repos/private/panel/panel/reactive.py", line 477, in _comm_event
    self._process_bokeh_event(doc, event)
  File "/home/jovyan/repos/private/panel/panel/reactive.py", line 414, in _process_bokeh_event
    self._process_event(event)
  File "/home/jovyan/repos/private/panel/panel/widgets/button.py", line 243, in _process_event
    self.clicks += 1
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 367, in _f
    instance_param.__set__(obj, val)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 369, in _f
    return f(self, obj, val)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/__init__.py", line 625, in __set__
    super(Dynamic,self).__set__(obj,val)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 369, in _f
    return f(self, obj, val)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 1252, in __set__
    obj.param._call_watcher(watcher, event)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 2043, in _call_watcher
    self_._execute_watcher(watcher, (event,))
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 2025, in _execute_watcher
    watcher.fn(*args, **kwargs)
  File "/home/jovyan/repos/private/panel/panel/widgets/chat.py", line 1207, in _click_send
    self.send(value=value, user=self.user, avatar=self.avatar, respond=True)
  File "/home/jovyan/repos/private/panel/panel/widgets/chat.py", line 878, in send
    entry = self._build_entry(value, user=user, avatar=avatar)
  File "/home/jovyan/repos/private/panel/panel/widgets/chat.py", line 703, in _build_entry
    entry = ChatEntry(**value, **entry_params)
  File "/home/jovyan/repos/private/panel/panel/widgets/chat.py", line 283, in __init__
    ParamMethod(self._render_value, **render_kwargs),
  File "/home/jovyan/repos/private/panel/panel/param.py", line 788, in __init__
    self._replace_pane()
  File "/home/jovyan/repos/private/panel/panel/param.py", line 854, in _replace_pane
    new_object = self.eval(self.object)
  File "/home/jovyan/repos/private/panel/panel/param.py", line 813, in eval
    return eval_function(function)
  File "/home/jovyan/repos/private/panel/panel/util/__init__.py", line 325, in eval_function
    return function(*args, **kwargs)
  File "/home/jovyan/repos/private/panel/.venv/lib/python3.10/site-packages/param/parameterized.py", line 407, in _depends
    return func(*args, **kw)
  File "/home/jovyan/repos/private/panel/panel/widgets/chat.py", line 417, in _render_value
    value_panel.css_classes = [*value_panel.css_classes, "message"]
AttributeError: 'NoneType' object has no attribute 'css_classes
```

Reply:
This was because dataframe_renderer was returning layout.insert() which returns None.